Azure-Samples/azure-intelligent-edge-patterns
# Adding details and helper scripts for Kubeflow installation on Azure Stack

## Purpose
* Add more details and address feedback from the adopters.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
```
Go to `/azure-intelligent-edge-patterns/Research/kubeflow-on-azure-stack/` and follow the instructions. For example, at a master node of Azure Stack Kubernetes cluster fun script `kubeflow_install.sh` and other steps Readme.md describes, and see that you are able to create a Kubeflow cluster in that environment.

```

## What to Check
* check that the instructions are simple enough for a data scientist to follow and get a Jupyter Server available.
